test/service/test_response.py

```diff
@@ -2372,7 +2372,7 @@ class TestSchemaTestDataCannedBundle(WebServiceTestCase):
     def bundles(cls) -> list[BundleFQID]:
         return [
             # Bundles for project 90bf705c
-            # https://github.com/HumanCellAtlas/schema-test-data/tree/2a62a7f4
+            # https://github.com/HumanCellAtlas/schema-test-data/tree/eb8f7d34
```

Comment:
Either use the long ref consistently or the short one but don't mix. I'd prefer the short one, if that's possible.